python/graphstorm/gsf.py

```diff
@@ -141,10 +149,32 @@ def initialize(ip_config=None, backend='gloo', local_rank=0, use_wholegraph=Fals
     use_wholegraph: bool
         Whether to use wholegraph for feature transfer.
         Default: False.
+    use_graphbolt: bool
+        Whether to use GraphBolt graph representation.
+        Default: False.
     """
-    # We need to use socket for communication in DGL 0.8. The tensorpipe backend has a bug.
-    # This problem will be fixed in the future.
-    dgl.distributed.initialize(ip_config, net_type='socket')
+    dgl_version = version('dgl')
+    if int(dgl_version.split('.')[0]) > 1:
+        dgl.distributed.initialize(
+            ip_config,
+            net_type='socket',
+            use_graphbolt=use_graphbolt,
+        )
+    else:
+        if use_graphbolt:
+            logging.warning(
+                (
+                    "use_graphbolt was 'true' but but DGL version was %s. "
+                    "GraphBolt requires DGL version >= 2.x."
+                ),
+                dgl_version
+                )
+        # We need to use socket for communication in DGL 0.8. The tensorpipe backend has a bug.
```

Comment:
Why does the comment mean?

Reply:
This was pre-existing I just moved it. If we don't need it any more I'll remove it.